### DISCUSSION TITLE
Once received module - Allow good host

### DISCUSSION THREAD

**Discussion by srsr** [rspamd/rspamd#5184] [Q&A] [answered]
Hello,
I'm wondering how to add a good host to exclude from check.
I setup good_host in /etc/rspamd/override.d/once_received.conf like this : 
```
once_received {
    good_host = "^www.myserver.fr";
}
```
Does I misunderstand the documentation? ( [https://www.rspamd.com/doc/modules/once_received.html](https://www.rspamd.com/doc/modules/once_received.html) )
I also try to remove the '^' character without success
Best regards

**Comment by fatalbanana** [comment, marked as answer]
> I setup good_host in /etc/rspamd/override.d/once_received.conf like this :

The settings shouldn't be wrapped in `once_received` in this context, they are already inside module configuration.

**Comment by srsr** [comment]
Thanks a lot fatalbanana! :)
